client/rpc-java-client/src/main/java/org/sonarsource/sonarlint/core/rpc/client/SonarLintRpcClientImpl.java

```diff
@@ -89,8 +88,8 @@ public SonarLintRpcClientImpl(SonarLintRpcClientDelegate delegate, ExecutorServi
     this.requestAndNotificationsSequentialExecutor = requestAndNotificationsSequentialExecutor;
   }
 
-  protected <R> CompletableFuture<R> requestAsync(Function<CancelChecker, R> code) {
-    CompletableFuture<CancelChecker> start = new CompletableFuture<>();
+  protected <R> CompletableFuture<R> requestAsync(Function<org.eclipse.lsp4j.jsonrpc.CancelChecker, R> code) {
+    CompletableFuture<org.eclipse.lsp4j.jsonrpc.CancelChecker> start = new CompletableFuture<>();
```

Comment:
Maybe we don't need to use the FQN here, as our own cancel checker is called differently. But if you think it makes things clearer, I'm fine keeping it